p2p/conn/connection_test.go

```diff
@@ -189,7 +189,7 @@ func TestMConnectionPongTimeoutResultsInError(t *testing.T) {
 	}()
 	<-serverGotPing
 
-	pongTimerExpired := mconn.config.PongTimeout + 20*time.Millisecond
+	pongTimerExpired := mconn.config.PongTimeout + 200*time.Millisecond
```

Comment:
Generously extended timeouts to fix:

--- FAIL: TestTransportMultiplexAcceptNonBlocking (0.08s)
##[error]    transport_test.go:331: connection failed: Fast peer timed out
FAIL
FAIL	github.com/tendermint/tendermint/p2p	5.160s

--- FAIL: TestTransportMultiplexAcceptNonBlocking (0.08s)
##[error]    transport_test.go:331: connection failed: read tcp 127.0.0.1:43790->127.0.0.1:33181: i/o timeout
FAIL
FAIL	github.com/tendermint/tendermint/p2p	5.036s

--- FAIL: TestMConnectionPongTimeoutResultsInError (0.20s)
    connection_test.go:198: Expected to receive error after 65ms
FAIL
FAIL	github.com/tendermint/tendermint/p2p/conn	2.492s